.travis.yml

```diff
@@ -1,36 +1,45 @@
 language: python
-sudo: false
-cache: pip
+cache:
+  directories:
+  - $HOME/.cache/pip
+  - $HOME/.cache/pre-commit
 
 python:
-  - "3.5"
+  - "3.6"
 
 addons:
   postgresql: "9.6"
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
-      - python-lxml # because pip installation is slow
+
+stages:
+  - linting
+  - test
+
+jobs:
+  include:
+    - stage: linting
+      name: "pre-commit"
+      before_install:
+      install: pip install pre-commit
+      script: pre-commit run --all --show-diff-on-failure
+      after_success:
+    - stage: test
+      env:
+        - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+    - stage: test
+      env:
+        - TESTS="1" ODOO_REPO="OCA/OCB"
 
 env:
   global:
   - VERSION="13.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
 
-  matrix:
-  - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
-  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
-
-
 install:
-  - pip install -q unidecode
-  - pip install unicodecsv
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
```

Comment:
```suggestion
  - travis_install_nightly
  # Requirements to test server_environment modules
  - printf '[options]\n\nrunning_env = dev\n' > ${HOME}/.openerp_serverrc
```

Reply:
I have to check if now it's odoo_serverrc instead.

Reply:
I confirm it didn't change https://github.com/OCA/maintainer-quality-tools/blob/master/travis/test_server.py#L270